instant/flags.vim

```diff
@@ -51,3 +51,11 @@ call s:plugin.Flag('clang_format_style', 'file')
 " "goimports" (https://godoc.org/golang.org/x/tools/cmd/goimports) to
 " additionally adjust imports when formatting.
 call s:plugin.Flag('gofmt_executable', 'gofmt')
+
+""
+" The path to the js-beautify executable.
+call s:plugin.Flag('js_beautify_executable', 'js-beautify')
+
+""
+" Default formatter for javascript files, 'clang_format' or 'js_beautify'.
+call s:plugin.Flag('javascript_formatter', 'clang_format')
```

Comment:
That solves the immediate problem, but I wonder if there is something more general we should be thinking about.  We are going to get more formatters that overlap like this (we have multiple java formatters at google).  Putting a new flag for every one of them would work, but doesn't seem like the best solution.  I am trying to think if we can generalize in some fashion...